scripts/o3de/o3de/register.py

```diff
@@ -917,8 +982,11 @@ def add_parser_args(parser):
     parser.add_argument('-r', '--remove', action='store_true', required=False,
                         default=False,
                         help='Remove entry.')
-    parser.add_argument('-f', '--force', action='store_true', default=False,
+    group = parser.add_mutually_exclusive_group(required=False)
+    group.add_argument('-f', '--force', action='store_true', default=False,
                         help='For the update of the registration field being modified.')
```

Comment:
No part of the change, but it looks like this was meant to read
```suggestion
                        help='Force the update of the registration field being modified.')
```